transitmatters/t-performance-dash
# Delivered Service

## Motivation

Add a graph that shows % of scheduled service delivered
Also a graph for % of baseline delivered

## Changes
![Screen Shot 2023-05-19 at 12 29 35 PM](https://github.com/transitmatters/t-performance-dash/assets/46229349/e34e08df-8d8b-405c-a5e3-dafd5a2c4cd2)


## Testing Instructions

Spot-check the numbers to ensure they seem right.

I am going to manually deploy this (5/19/23 @ 12:30)
